packages/services/api/src/modules/schema/providers/schema-helper.ts

```diff
@@ -37,7 +37,7 @@ export function ensureSingleSchema(schema: Schema | Schema[]): SingleSchema {
   throw new Error('Expected a single schema');
 }
 
-export function ensureCompositeSchemas(schemas: readonly Schema[]): CompositeSchema[] | never {
```

Comment:
I assumed this `never` was because the function threw an array, but it only filters. So I'm not sure why this never is here... removed.